### ISSUE TITLE
uninitialized constant Hashie::Extensions::Array from the rake task

### ISSUE THREAD

**Issue by Cabbit** [jhollinger/otr-activerecord#6]
When using the following rake file

```
require 'bundler/setup'
require 'rake/testtask'
load 'tasks/otr-activerecord.rake'

namespace :db do
  task :environment do
    require_relative 'config/application'
  end
end

Rake::TestTask.new do |t|
  t.libs << 'test'
  t.test_files = FileList['spec/**/*_spec.rb']
end
```

I get the following error message, when trying to run the example applications on Ruby 2.3.1

```
rake aborted!
NameError: uninitialized constant Hashie::Extensions::Array
Did you mean?  Hashie::Array
               Array
/usr/local/bundle/gems/hashie-3.4.5/lib/hashie/array.rb:3:in `<class:Array>'
/usr/local/bundle/gems/hashie-3.4.5/lib/hashie/array.rb:2:in `<module:Hashie>'
/usr/local/bundle/gems/hashie-3.4.5/lib/hashie/array.rb:1:in `<top (required)>'
/usr/local/bundle/gems/hashie-3.4.5/lib/hashie/mash.rb:2:in `require'
/usr/local/bundle/gems/hashie-3.4.5/lib/hashie/mash.rb:2:in `<top (required)>'
/usr/local/bundle/gems/hashie-forbidden_attributes-0.1.1/lib/hashie-forbidden_attributes/hashie/mash.rb:1:in `require'
/usr/local/bundle/gems/hashie-forbidden_attributes-0.1.1/lib/hashie-forbidden_attributes/hashie/mash.rb:1:in `<top (required)>'
/usr/local/bundle/gems/hashie-forbidden_attributes-0.1.1/lib/hashie-forbidden_attributes.rb:1:in `require'
/usr/local/bundle/gems/hashie-forbidden_attributes-0.1.1/lib/hashie-forbidden_attributes.rb:1:in `<top (required)>'
/usr/local/bundle/gems/otr-activerecord-1.2.0/lib/otr-activerecord.rb:2:in `require'
/usr/local/bundle/gems/otr-activerecord-1.2.0/lib/otr-activerecord.rb:2:in `<top (required)>'
/usr/local/bundle/gems/otr-activerecord-1.2.0/lib/tasks/otr-activerecord.rake:5:in `require'
/usr/local/bundle/gems/otr-activerecord-1.2.0/lib/tasks/otr-activerecord.rake:5:in `<top (required)>'
/var/www/bunny/rakefile:3:in `load'
/var/www/bunny/rakefile:3:in `<top (required)>'
/usr/local/bundle/gems/rake-11.2.2/exe/rake:27:in `<top (required)>'
(See full trace by running task with --trace)

```

**Comment by jhollinger** [comment]
Sounds like `hashie` made some breaking internal changes. god i hate that gem. Maybe try bumping `hashie` down a couple versions in your `Gemfile.lock`. Meantime I'll try to find a way to remove the dependency on `hashie-forbidden_attributes`.

**Comment by jhollinger** [comment]
Yeah this just seems like a bug in Hashie 3.4.5. Looks like it might be fixed in Hashie 3.4.6.
